ruby_on_rails/appsignal.md

```diff
@@ -34,6 +34,46 @@ Message format: **logfmt**
 
 Then add this log drain using the heroku commands displayed.
 
+## Correct Severity
+
+According to the docs, getting the severity to be anything but "INFO" is not possible using the heroku drain.
+
+However, there is now a way to send the `"severity=XYZ"` logfmt information and have that be applied correctly in appsignal. Unfortunately, just setting this seems to break the recognition of `request_id` in the format `[1234-5678]`. So we have to override the `ActiveSupport::TaggedLogging::Formatter` to add both the `severity` and the `request_id` in logfmt syntax.
```

Comment:
In the herold project I saw that you attempted to revert back to the less invasive logger overide? So this didn't work out I guess? https://github.com/renuo/herold-planner/commit/9f2ab8eeceeacd0e2975f89e3ca2f73239462183

Reply:
No, it doesn't parse the `request_id` anymore using that setup.